jar/src/cljc/sixsq/slipstream/client/api/utils/http_utils.cljc

```diff
@@ -0,0 +1,16 @@
+(ns sixsq.slipstream.client.api.utils.http-utils)
+
+(def http-lib-insecure-key :kvlt.platform/insecure?)
```

Comment:
You might want to add the `^:const` flag to avoid an extra var redirection in the lookups.